### ISSUE TITLE
Implement tax deduction of private health insurance contributions

### ISSUE THREAD

**Issue by ChristianZimpelmann** [iza-institute-of-labor-economics/gettsim#269]
### Current and desired situation

We currently do not consider that contributions for a private health insurance can be deducted. This is especially relevant for civil servants.

### Proposed implementation

Add new input variable `prv_krankenv_beitr_m` and adjust `vorsorge/vorsorge_ab_2020` accordingly.

### Considered alternatives

-

**Comment by hmgaudecker** [comment]
Great.

We need to be very careful when measuring this, though. There is a distinction between the fees paid and what can be deducted. The latter comprises fees for "required" medical needs, excluding special treatment by chief physician, single rooms in hospital, etc. 

Not sure how to go about it. Surveys will reasonably measure monthly contributions. Tax Payer Panel would have what can be  deducted from taxes. 

We need only the amount that can be deducted. Maybe include that and at some point a try to supply an imputation factor for survey data input of total contributions?

**Comment by Eric-Sommer** [comment]
> Not sure how to go about it. Surveys will reasonably measure monthly contributions. Tax Payer Panel would have what can be deducted from taxes.
>
> We need only the amount that can be deducted. Maybe include that and at some point a try to supply an imputation factor for survey data input of total contributions?

I am not sure whether this should be in the scope of gettsim. What we can do right, we can do, such as implementing the deduction rule. Imputations can be done worse or better, but are inevitably imperfect. I am not sure we want to do this right now. We can do it of course for our own gettsim applications, but IMO it should not be part of core gettsim any time soon.

**Comment by hmgaudecker** [comment]
Agreed this would be in convenience functions and not in the core, sorry if
that was not clear.

My main point is that it i needs to be very  clear from the variable name
that this cannot be simply the payments you may see in the SOEP/EVS/...

Eric-Sommer <notifications@github.com> schrieb am Fr., 12. Feb. 2021, 08:12:

**Comment by hmgaudecker** [comment]
Also see #205 for broader discussion.
